harness/reflect.go

```diff
@@ -433,7 +433,8 @@ func appendAction(fset *token.FileSet, mm methodMap, decl ast.Decl, pkgImportPat
 			var importPath string
 			typeExpr := NewTypeExpr(pkgName, field.Type)
 			if !typeExpr.Valid {
-				return // We didn't understand one of the args.  Ignore this action. (Already logged)
+				log.Printf("Didn't understand argument '%s' of action %s. Ignoring.\n", name, getFuncName(funcDecl))
```

Comment:
It appears to be inconsistent, the lines at the bottom use revel logging

https://github.com/roblillack/revel-cmd/blob/feature/improve-route-generation-warnings/harness/reflect.go#L769-L777